### DISCUSSION TITLE
GUI Enhancement: Buttons on top

### DISCUSSION THREAD

**Discussion by webdb22** [jhuckaby/Cronicle#646] [Ideas]
I came across Cronicle only recently, I'm very impressed how straight forward, functional and reliable it is, awesome work @jhuckaby!

What I've missed when creating the first few Events, Categories, etc., are buttons on top of the pages instead of at the bottom. E.g. on the "Schedule" tab, if there are already some Events scheduled, you need to scroll down to the very bottom of the page to click "Add Event...". Or when editing a Plugin, you need to scroll down to click "Copy Plugin...".

To summarize: It would be great to have additionally buttons on the top of the pages to have direct access without the need of scrolling down.

Thanks in advance for considering this idea!

**Comment by mikeTWC1984** [comment]
Agree, scrolling down to click the new event button is a bit of annoying sometimes. Probably that button could be just replicated as an icon search widget.

**Comment by jhuckaby** [comment]
Thank you for the feature request!  This is actually already solved in Cronicle v2 (Orchestra), which is due for release later this year.

**Reply by webdb22** [reply]
@jhuckaby Thanks for your prompt reply, happy to hear this is already resolved in V2, so looking forward to "Orchestra"!
